.resinci.yml

```diff
@@ -0,0 +1,4 @@
+---
+npm:
+  platforms: []
```

Comment:
We have the objective of replacing Circle CI with balena CI, but I do not propose to it in this PR. Given that the test code in this repo is not setup for balena CI, the default balena CI tests always fail (see past PRs). This change merely tells balena CI not to bother to run tests that we know will fail anyway.